src/game_api/script/usertypes/entities_activefloors_lua.cpp

```diff
@@ -120,6 +118,8 @@ void register_usertypes(sol::state& lua)
         sol::base_classes,
         sol::bases<Entity, Movable>());
 
+    falling_platform["emitted_light"] = &FallingPlatform::timer; // wrong name, kept for backwards compatibility
```

Comment:
When a mistake is this obvious, I think it's the users fault for using it and not saying anything, so we can break backwards compatibility.